gui.py

```diff
@@ -64,17 +65,19 @@ def init_widgets(self):
         # Directory information
         directory_frame = ttk.Frame(input_frame, borderwidth=3, relief=tk.GROOVE)
         directory_frame.pack()
-        delta_dir_tt = ("Contains paths - '{Base Scenario}/trfl{Base Year}"
-                        "{Base Scenario}.dat' and '{Future Scenario}/"
-                        "[tav_/tmfs/trfl]{Future Year}{Future Scenario}.dat'")
+        delta_dir_tt = ("Contains folders containing the TELMoS planning data csv files "
+                        "and goods dat files. Folders and files should be named according to the "
+                        "year and scenario code.")

```

Comment:
I've added examples into the tooltips, although this does lead to them being quite long. Not sure if there is a better way of showing an example.

Reply:
Hmm. Could it possibly be a text box rather than a tooltip then? To be honest, I don't think it's particularly obvious that there is a tooltip, so if we're going to stick with that it might be better to have e.g. a question mark by each section that has the tooltip connected. Doesn't necessarily matter for this PR though, so maybe we mark it as resolved for now and discuss in a new Issue?